src/object/assign.js

```diff
@@ -0,0 +1,19 @@
+import assign from 'lodash/fp/assign'
+import { lodashFpConvertOptions } from '../core/consts'
+import update from '../core/update'
+
+const rawAssign = assign.convert(lodashFpConvertOptions)
+
+/**
+ * Applies <code>!</code> to the property.
```

Comment:
Rather "Assigns own enumerable string keyed properties of source objects to the destination object. Source objects are applied from left to right. Subsequent sources overwrite property assignments of previous sources.".